PiggyBank.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <link rel="stylesheet" type="text/css" href="PiggyBank.css" media="screen" />
+    <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js" type="text/javascript"></script>
+    <script src="PiggyBank.js"></script>
```

Comment:
Although your PiggyBank.js is written inside of a $(document).ready( () => {} ), I would still suggest including your jQuery scripts at the bottom of the body just before you close the </body> tag.  This will help avoid situations where you're targeting DOM elements that haven't rendered to the page yet.  Again, you have it covered with the .ready() listener, but it is not always the case that your jQuery lives inside the .ready()